public/js/user-actions.js

```diff
@@ -13,13 +16,56 @@ function UserActions(localDispatcher) {
 }
 
 
-UserActions.prototype.setCurrentUser = function(user) {
-  console.log('UserActions: setting current user', user);
-  this.dispatcher.dispatch({
-    actionType: 'set-user',
-    user: user,
-  });
-};
+UserActions.prototype = assign({}, {
+
+  signIn: function(accessToken, opt) {
+    opt = opt || {};
+    if (!opt.jquery) {
+      opt.jquery = $;
+    }
```

Comment:
I'm sure there's logic behind it, but passing jQuery around like this feels like an anti-pattern. Shouldn't it be `require`able?